### PR TITLE
Report the config via the JSON Server Stats

### PR DESCRIPTION
**Pull request by peternewman**

### CODE REVIEW

File: include/olad/Preferences.h [hunk 1]
```diff
@@ -327,6 +327,12 @@ class PreferencesFactory {
    */
   virtual Preferences *NewPreference(const std::string &name);
 
+  /**
+   * @brief The location where preferences will be stored.
+   * @return the location
+   */
+  virtual std::string Source() const = 0;
```
Comment (nomis52) [inline include/olad/Preferences.h]: Maybe call this "ConfigLocation" or something similar.

Comment (peternewman) [inline include/olad/Preferences.h]: Can do, I was just matching the Preferences class name.

Comment (peternewman) [inline include/olad/Preferences.h]: Fixed

**Comment by peternewman** [comment]
This should be good to go when Travis is done if you're happy.

**Pull request merged**